AutomaticStockTrader.Core/ServiceCollectionExtentions.cs

```diff
@@ -15,16 +15,26 @@
 using Environments = Alpaca.Markets.Environments;
 using Quartz;
 using Quartz.Impl.Calendar;
+using Microsoft.Extensions.Configuration;
+using AutomaticStockTrader.Repository.Configuration;
 
 namespace AutomaticStockTrader.Core
 {
     public static class ServiceCollectionExtentions
     {
         private const string EST_STANDARD_NAME = "Eastern Standard Time";
 
-        public static IServiceCollection AddStockAutoTrading(this IServiceCollection services, AlpacaConfig config)
+        public static IServiceCollection AddStockAutoTrading(this IServiceCollection services, IConfiguration config)
         {
+            services
+                .AddOptions()
+                .Configure<AlpacaConfig>(config)
```

Comment:
future: this project will likely benefit from config validation on startup

https://andrewlock.net/adding-validation-to-strongly-typed-configuration-objects-in-asp-net-core/